docs/application/tizen-studio/faq/faq.md

```diff
@@ -0,0 +1,180 @@
+# Frequently Asked Questions 
+
+This page answers to the most commonly asked questions or errors that you may encounter while installing Tizen studio and developing apps in Tizen Studio.
```

Comment:
Does this sound better?
This page answers to the most commonly asked questions or errors that you may encounter while installing or developing apps in Tizen Studio.